packages/docs/src/components/content/PropTableHtmlElementRow.tsx

```diff
@@ -0,0 +1,39 @@
+import humanizeList from 'humanize-react';
```

Comment:
Could you use the [`Intl` API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) to handle this instead of importing a new library?

Reply:
😍

Reply:
Oh wait, the reason I used a library is because these are actually arrays of React elements and not strings. Each element in an array of React elements needs a key, so this library will take an array of React elements and return one where more React elements are inserted for the commas and conjunctions with keys without having to convert the links and code blocks into strings and then use `dangerouslySetInnerHTML` later.

Reply:
Oh fiddlesticks! Didn't know that! I'm good with the library then.